filebeat/module/squid/access/_meta/fields.yml

```diff
@@ -0,0 +1,45 @@
+- key: access
+  type: group
+  description: >
+    squid Module
+  fields:
+    - name: duration
```

Comment:
This needs completion.

Reply:
I have added test log

Reply:
Could you also update the description of the fields.

Looking at your ingest pipeline below it seems it does not put the fields into the same fields as defined here. You must be aware that all the fields should be prefixed by `squid.access` if they don't overlap with some global fields.